src/bessel.jl

```diff
@@ -337,7 +337,7 @@ end
 
 besselj(nu::Cint, x::Float64) = ccall((:jn, libm), Float64, (Cint, Float64), nu, x)
 besselj(nu::Cint, x::Float32) = ccall((:jnf, libm), Float32, (Cint, Float32), nu, x)
-
+besselj(nu::Cint, x::Float16) = besselj(nu, Float32(x))
```

Comment:
The convention is to convert the result back to `Float16`.

Reply:
Oops! Forgot that.  In a minute.